core/src/main/scala/org/http4s/Message.scala

```diff
@@ -273,6 +274,34 @@ final class Request[F[_]](
 
   def queryString: String = uri.query.renderString
 
+  /** cURL representation of the request.
+    *
+    * Supported cURL-Parameters are: -X, -H
+    *
+    */
+  def asCurl(
+      redactHeadersWhen: CaseInsensitiveString => Boolean = Headers.SensitiveHeaders.contains)
+      : String = {
+
+    /*
+     * escapes characters that are used in the curl-command, such as '
+     */
+    def escapeQuotationMarks(s: String) = s.replaceAll("'", """'\\''""")
```

Comment:
This adds a set of single quotation marks around the escaped one. Is that on purpose?

```scala
scala> escapeQuotationMarks("Can't do this?")
res2: String = Can'\''t do this?
```